pkgs/npm/src/cli.ts

```diff
@@ -303,6 +303,13 @@ function extractCLIWrapperArgs(args: string[]): {
   return { cliParams, outArgs };
 }
 
+function getEndNameFromPath(pathLine: string): string {
+  const pathParts = pathLine.split(path.sep);
+  const executableName = pathParts[pathParts.length - 1];
+
+  return executableName.split(".")[0];
```

Comment:
is this to cut of `.exe`? add a comment?

Reply:
yes